CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 This repository contains the `FreeRTOS AWS Reference Integrations`, which are pre-integrated FreeRTOS projects that demonstrate connectivity with AWS IoT. The repository contains projects for many different microcontroller evaluation boards.
 
+## 202012.01 November 2021
+
+This is a minor version update to the source code for patching Espressif ESP-IDF V3.3, for mitigating Bluetooth vulnerability named Braktooth(https://asset-group.github.io/disclosures/braktooth). The changes are contained within the espressif IDF SDK submodule and the associated ESP-IDF components used within this repository.
```

Comment:
can we explicit about "associated ESP-IDF components"? better if we says which platform is impacted by this change.